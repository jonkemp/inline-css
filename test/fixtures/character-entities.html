<html><link rel="stylesheet" href="character-entities.css" /><body>
<p>Dear &lt;insert name&gt;,<insert name=""></insert></p></body></html>
