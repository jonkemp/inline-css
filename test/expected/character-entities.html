<html>
<head></head>
<body>
<p style="margin: 0 0 12px 0; margin-bottom: 12px;">Dear &lt;insert name&gt;,<insert name=""></insert></p>
</body></html>
